engines/ags/lib/std/functional.h

```diff
@@ -48,6 +48,10 @@ struct function {
 	operator _Fty &() {
 		return *_fn;
 	}
+
+	operator bool() {
```

Comment:
```suggestion
	operator bool() const {
```
It's probably a good idea to declare it as const so that conversion to `bool` also works as expected if you have a `const std::function`.